core/controllers/base.py

```diff
@@ -125,10 +134,11 @@ def _clear_login_cookies(response_headers):
 
 
 class LogoutPage(webapp2.RequestHandler):
+    """ Class which handles the logout logic"""
 
     def get(self):
-        """Logs the user out, and returns them to a specified page or the home
-        page.
+        """Logs out the user/developer, and returns them to a specified
```

Comment:
Logs the user out, and returns them to a specified page (or the home page if no follow-up page is specified).